source/features/one-click-review-submission.tsx

```diff
@@ -3,16 +3,15 @@ import delegate, {DelegateEvent} from 'delegate-it';
 import * as pageDetect from 'github-url-detection';
 import CheckIcon from 'octicons-plain-react/Check';
 import FileDiffIcon from 'octicons-plain-react/FileDiff';
+import {expectElement} from 'select-dom';
 
 import features from '../feature-manager.js';
 import observe from '../helpers/selector-observer.js';
 import {assertNodeContent} from '../helpers/dom-utils.js';
 
 function replaceCheckboxes(originalSubmitButton: HTMLButtonElement): void {
 	const form = originalSubmitButton.form!;
-	const actionsRow = originalSubmitButton.closest('.Overlay-footer');
-	// TODO: For GHE. Remove after June 2024
-	const legacyActionsRow = originalSubmitButton.closest('.form-actions')!;
+	const actionsRow = originalSubmitButton.closest('.Overlay-footer')!;
```

Comment:
Looks like dropping this has broken GHE 3.12.3, I get `actionRow is null` now. Where'd we get the comment from?

Reply:
Feel free to revert changes to this file with an updated 2025 comment. It's weird that GitHub hasn't updated the DOM at all

Reply:
Please open a PR